skill/__init__.py

```diff
@@ -0,0 +1,130 @@
+from neon_utils.skills.neon_skill import NeonSkill, LOG
+from mycroft_bus_client import Message
+import os
+import json
+from .answer_checks import Check
+
+class InstructionsSkill(NeonSkill):
+
+    def __init__(self):
+        '''
+        lang: ISO 639-1
+        '''
+        super(InstructionsSkill, self).__init__(name="InstructionsSkill")
+        basepath = os.path.dirname(os.path.realpath(__file__))
+        json_path = basepath+'/instructions/en/demo2_en.jsonl'
+        self.json_path = json_path
+        self.Check = None
+        # self.lang = 'en'
+        self.json_path = json_path
+        self.answer_list = []
+        self.question_id = '1'
+        self.words_from_prev_answer = ''
+
+    def initialize(self):
+        self.register_intent_file("run_instructions.intent", self.handle_instructions)
+
+        # When first run or demo prompt not dismissed, wait for load and prompt user
+        if self.settings['prompt_on_start'] and not self.server:
```

Comment:
You should add a `settingsmeta.yml` file with this setting and default value, like in the demo skill: https://github.com/NeonGeckoCom/skill-demo/blob/master/settingsmeta.yml